tensorflow/compiler/mlir/xla/tests/hlo-legalize-to-lhlo.mlir

```diff
@@ -11,6 +11,17 @@ func @attrs_copy(%operand: memref<2x2xf32>, %result: memref<2x2xf32>) {
   return
 }
 
+// CHECK-LABEL: func @func_op
```

Comment:
Could you add a test with multiple operations? And also one where a value is used in multiple places, for instance in a further op and return?